pandas/tests/frame/indexing/test_where.py

```diff
@@ -653,3 +653,22 @@ def test_where_categorical_filtering(self):
         expected.loc[0, :] = np.nan
 
         tm.assert_equal(result, expected)
+
```

Comment:
can you move these to the generic EA indexing tests

Reply:
this file is the canonical place for DataFrame.where tests

Reply:
ok, worth parameterizing over dtype then? this is *just* testing a single EA type

Reply:
not worth worrying about ATM, hopefully will get a solution before _too_ long that avoids object-dtype cast